parm/gdas/atm_lgetkf_bkg_staging.yaml.j2

```diff
@@ -0,0 +1,27 @@
+{% set ftype_list = ['fv_core.res', 'fv_srf_wnd.res', 'fv_tracer.res', 'phy_data', 'sfc_data'] %}
+{% set prefix = current_cycle | to_fv3time %}
```

Comment:
We should build in some looping capability for over multiple times when we have FGAT.